### ISSUE TITLE
Helm chart archive file linked from GitHub Releases page has "dirty" in the tag

### ISSUE THREAD

**Issue by YutaHiguchi-bsn** [flannel-io/flannel#1805]
Helm chart archive file linked from GitHub Releases page: https://github.com/flannel-io/flannel/releases/download/v0.22.2/flannel.tgz
has -dirty in the tag.

Specifically in flannel/values.yaml:
```yaml
flannel:
  # kube-flannel image
  image:
    repository: docker.io/flannel/flannel
    tag: v0.22.2-dirty
```

Source archive or the tagged commit does not have this issue
https://github.com/flannel-io/flannel/blob/148b6d6a58deb864519bbc474f4c531a0799e910/chart/kube-flannel/values.yaml#L8-L12

File from the previous release did not have this problem.
If the release process has changed since the last version, there might be some issue in the new steps?
https://github.com/flannel-io/flannel/releases/download/v0.22.1/flannel.tgz

## Expected Behavior
tag to not have -dirty suffix matching the tagged source code

## Current Behavior
flannel/values.yaml in the archive contains a tag with -dirty suffix

**Comment by manuelbuil** [comment]
After merging 3 PRs, we will release v0.22.3 in a few days and then hopefully the chart will be correct

**Comment by rbrtbnfgl** [comment]
I removed the dirty flag from the chart published.

**Comment by YutaHiguchi-bsn** [comment]
Thank you both, v0.22.2 seems to be fixed and also the newer version v0.22.3 doesn't have the dirty flag as well.
